src/views/Home/index.tsx

```diff
@@ -85,11 +115,31 @@ const HomeView = (): JSX.Element => {
       const response = await getPrefabs(query)
 
       if (response.data && response.data.success) {
-        const categories = getCategories(response.data.data)
+        setCategories(getCategories(response.data.data))
+
+        if (selectedValues.length > 0) {
+          const selectedPacks = selectedValues.map(x => x['value'] as string);
```

Comment:
any chance providing the type to the object beforehand without the need to do type coercion?

Reply:
I'll see if I can do that, was getting errors about the type being void with this one before